pkg/proto/configuration/bb_storage/bb_storage.proto

```diff
@@ -38,4 +38,15 @@ message StorageConfiguration {
 
   // List of instances which can upload to the Action Cache.
   repeated string allow_ac_updates_for_instances = 6;
+
+  // TLS configuration, if enabled.
+  TLSConfiguration tls = 7;
+}
+
+// Used for both GRPC and HTTP listeners.
+message TLSConfiguration {
```

Comment:
Looks like this should be in a separate proto, so it can be used from the other components.
Will extract it.